1.The time required for servicing transmissions is normally distributed with  = 45 minutes and  = 8 minutes.The service
manager plans to have work begin on the transmission of a customer’s car 10 minutes after the car is dropped off and the
customer is told that the car will be ready within 1 hour from drop-off. What is the probability that the service manager 
cannot meet his commitment? 

In [1]:
from scipy import stats 
from scipy.stats import norm 

In [3]:
stats.norm.cdf(0.625)


0.7340144709512995

In [4]:
1-0.7340144709512995

0.26598552904870054

In [6]:
# finding z-score

z=(50-45)/8
z

0.625

In [8]:
# finding probablity
 
stats.norm.cdf(0.625)    

0.7340144709512995

In [9]:
1-stats.norm.cdf(0.625) 

0.26598552904870054

the probablity of service manager can meet his commitment is 0.265

2) The current age (in years) of 400 clerical employees at an insurance claims processing center is normally distributed with 
   mean  = 38 and Standard deviation  =6. For each statement below, please specify True/False. If false, briefly explain why.
   
   




In [12]:
# A.More employees at the processing center are older than 44 than between 38 and 44.

In [14]:
#p(X>44)

1-stats.norm.cdf(44,38,6)

0.15865525393145707

In [15]:
#p(38<X<44)

stats.norm.cdf(44,38,6)-stats.norm.cdf(38,38,6)

0.3413447460685429

In [16]:
# B. training program for employees under the age of 30 at the center would be expected to attract about 36 employees.

In [17]:
# p(X<30)

stats.norm.cdf(30,38,6)

0.09121121972586788

In [20]:
# total no of employee under the age 30 

# N*p(x<30)

400*(0.09121121972586788)

36.484487890347154

SO statement A is False.
p(X>44)>p(38<X<44)


3) If X1 ~ N(μ, σ2) and X2 ~ N(μ, σ2) are iid normal random variables, then what is the difference between 2 X1 and X1 + X2? 
Discuss both their distributions and parameters.  

slution :-  As we know that if  X ∼ N(µ1, σ1^2 ), and  Y ∼ N(µ2, σ2^2 ) are two independent  random variables then   X + Y ∼ N(µ1 + µ2, σ1^2 + σ2^2 ) , and      X − Y ∼ N(µ1 − µ2, σ1^2 + σ2^2 ) .
Similarly if  Z = aX + bY , where X and Y are as defined above, i.e Z is linear combination of X and Y , then Z ∼ N(aµ1 + bµ2,
a^2σ1^2 + b^2σ2^2 ).


  Therefore 
  
  2X1~ N(2 u,4 σ^2) and 
  
  X1+X2 ~  N(µ + µ, σ^2 + σ^2 ) ~ N(2 u, 2σ^2 ) 
  
  2X1-(X1+X2) = N( 4µ,6 σ^2)





4) Let X ~ N(100, 20^2). Find two values, a and b, symmetric about the mean, such that the probability of the random variable
taking a value between them is 0.99. 

In [21]:
stats.norm.interval(0.99,100,20)

(48.48341392902199, 151.516586070978)

5) Consider a company that has two different divisions. The annual profits from the two divisions are independent and have 
distributions Profit1 ~ N(5, 32) and Profit2 ~ N(7, 42) respectively. Both the profits are in $ Million. Answer the following
questions about the total profit of the company in Rupees. Assume that $1 = Rs. 45
A.	Specify a Rupee range (centered on the mean) such that it contains 95% probability for the annual profit of the company.
B.	Specify the 5th percentile of profit (in Rupees) for the company
C.	Which of the two divisions has a larger probability of making a loss in a given year?


In [27]:
import numpy as np

In [28]:
# finding total profit

In [33]:
total_profit = 5+7
print ("total profit is rs",total_profit * 45, "million")

total profit is rs 540 million


In [44]:
sd = np.sqrt((9)+(16))
SD=sd*45
SD

225.0

In [45]:
# A.	Specify a Rupee range (centered on the mean) such that it contains 95% probability for the annual profit of the company.

In [46]:
stats.norm.interval(0.95,540,225)

(99.00810347848784, 980.9918965215122)

the rupee range is between 99.00 to 980.99 million

B.	Specify the 5th percentile of profit (in Rupees) for the company

In [47]:
X= 540+(-1.645)*(225)

In [49]:
X

169.875

5th percentile profit is rs 170 million

C.	Which of the two divisions has a larger probability of making a loss in a given year?


In [51]:
# prob of division 1 loss p(x<0)
stats.norm.cdf(0,5,3)

0.0477903522728147

In [52]:
# prob od divon 2 loss p(x<0)

stats.norm.cdf(0,7,4)

0.040059156863817086